website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -160,6 +162,8 @@ The table below shows this endpoint's support for
 | ---------------- | ----------------- | ------------- | ------------ |
 | `YES`            | `all`             | `none`        | `acl:read`   |
 
+-> The corresponding CLI command is [`consul acl auth-method read`](https://www.consul.io/commands/acl/auth-method/read)
```

Comment:
@karl-cardenas-coding, @trujillo-adam : thoughts on whether:
- These sentences should end with a period?
- These sentences should be shown in an info box (blue background) or just as a normal sentence?

I don't think it warrants the info box highlight. (And I realize I contributed to this confusion by photoshopping in a blue highlight into my mock-ups ... next time I'll pick a different color to avoid potential confusion.) And if not in an info box, I think it should end in a period like a normal sentence. Want to confirm with you before asking @ChipV223 to make the edits though!

Reply:
Added a period during my later commits